src/test/common/test_crc32c.cc

```diff
@@ -265,3 +265,28 @@ TEST(Crc32c, RangeNull) {
     ASSERT_EQ(crc, *check);
   }
 }
+
+TEST(Crc32c, zeros_performance) {
+  utime_t start;
+  utime_t end;
+
+  for (size_t scale=1; scale < 31; scale++)
+  {
+    size_t size = (1<<scale) + rand()%(1<<scale);
+    start = ceph_clock_now();
+    uint32_t crc_a = ceph_crc32c(111, nullptr, size);
+    end = ceph_clock_now();
+    std::cout << "regular  method. size=" << size << " time= " << (double)(end-start)
+        << " at " << (double)size/(1024*1024)/(double)(end-start) << " MB/sec" << std::endl;
+
+    start = ceph_clock_now();
+    uint32_t crc_b = ceph_crc32c_sctp(111, nullptr, size);
```

Comment:
@kestrels "I would recommend you change this to calculate crc_b using ceph_crc32_func instead of ceph_crc32_sctp."
Good point, I have not noticed that last refactor allowed to do this.